network/network_windows.go

```diff
@@ -185,6 +190,58 @@ func (nm *networkManager) newNetworkImplHnsV1(nwInfo *NetworkInfo, extIf *extern
 	return nw, nil
 }
 
+// add ipv4 and ipv6 routes (if dual stack mode) to windows Node
+func (nm *networkManager) addNewNetRules(nwInfo *NetworkInfo) error {
```

Comment:
can you add detailed description on what this function does and what routes it configures?